### DISCUSSION TITLE
Removing Title Over Media Icons

### DISCUSSION THREAD

**Discussion by capritzl** [punxaphil/custom-sonos-card#100] [Q&A] [answered]
I want to remove the text that is displayed at the top of each media icon.  I just want the stations icon displayed without the white/gray text displaying the stations name.  How do I do that?

![image](https://user-images.githubusercontent.com/76876336/204841438-91681662-a4fd-4f00-948a-c4518daa8d7b.png)

**Comment by punxaphil** [comment, marked as answer]
You need to use styling
https://github.com/johanfrick/custom-sonos-card#css-styling

**Reply by capritzl** [reply]
I have modified the styling but I am not able to determine the setting to eliminate the station headers.

**Reply by punxaphil** [reply]
```
styles:
  media-button-title:
    display: none
```

**Comment by capritzl** [comment]
Worked great.
